CCIP: USDC guide (#1720)
smartcontractkit/documentation: src/content/ccip/tutorials/programmable-token-transfers.mdx

      1. In Remix IDE, under _Deploy & Run Transactions_, open the list of transactions of your smart contract deployed on _Ethereum Sepolia_.

>       1. Call the `allowlistDestinationChain` with <CopyText text="12532609583862916517" code/> as the destination chain selector, and <CopyText text="true" code/> as allowed. Each chain selector is found on the [supported networks page](/ccip/supported-networks).

Reviewer: ```suggestion
      1. Call the `allowlistDestinationChain`, setting the destination chain selector to <CopyText text="12532609583862916517" code/> and setting `allowed` to <CopyText text="true" code/>. Each chain selector is found on the [supported networks page](/ccip/supported-networks).
```
